apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/CharSequenceToContainCheckerSamples.kt

```diff
@@ -15,6 +15,26 @@ class CharSequenceToContainCheckerSamples {
         }
     }
 
+    @Test
+    fun atMost() {
+        expect("ABBC").toContain.atMost(2).value("B")
+
+        fails {
+            expect("AABBAA").toContain.atMost(3).value("A")
+        }
+    }
+
+    @Test
+    fun notOrAtMost() {
+        expect("ABBC").toContain.notOrAtMost(2).value("D")
+        expect("ABBC").toContain.notOrAtMost(2).value("B")
+
+        fails {
+            expect("AABBAA").toContain.notOrAtMost(3).value("A")
+            expect("AABDBAA").toContain.notOrAtMost(3).value("D")
+        }
```

Comment:
fails actually checks that your sample really fails. Therefore it would be better if we have those in two fails blocks so that we are sure that also the second fails:
```suggestion
        fails {
            expect("AABBAA").toContain.notOrAtMost(3).value("A")
        }
        fails {
            expect("AABDBAA").toContain.notOrAtMost(3).value("D")
        }
```

Reply:
My bad! I will correct it!